src/views/v2/tickets/submit/button-modal.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Block: Tickets
+ * Submit Button - Modal
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/tickets/submit/button-modal.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1amp
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ */
+
+/** @var \Tribe__Tickets__Attendee_Registration__Main $attendee_registration */
+$attendee_registration = tribe( 'tickets.attendee_registration' );
+
+if ( ! $attendee_registration->is_modal_enabled() ) {
+	return;
+}
+
+/* translators: %1$s: Event name, %2$s: Tickets label */
+$title = sprintf( _x( '%1$s %2$s', 'Tickets modal title.', 'event-tickets' ), get_the_title(), tribe_get_ticket_label_plural( 'event-tickets' ) );
+
+/* translators: %s: Tickets label */
+$button_text = sprintf( _x( 'Get %s', 'Get Tickets button.', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) );
+
+/**
+ * Allow filtering of the button classes for the tickets block.
+ *
+ * @since 4.11.3
+ *
+ * @param array $button_name The button classes.
+ */
+$button_classes = apply_filters(
+	'tribe_tickets_ticket_block_submit_classes',
+	[
+		'tribe-common-c-btn',
+		'tribe-common-c-btn--small',
+		'tribe-tickets__buy',
+	]
+);
+
+/**
+ * Filter Modal Content.
+ *
+ * @since 4.11.0
+ *
+ * @param string $content a string of default content.
+ * @param Tribe__Tickets__Editor__Template $template_obj the Template object.
+ */
+$content = apply_filters( 'tribe_events_tickets_attendee_registration_modal_content', '<p>Ticket Modal</p>', $this );
```

Comment:
Inherited.

> Is there a reason we're filtering this in the template?

Apparently because it is how the plugin populates the modal template. https://github.com/moderntribe/event-tickets/blob/ab3073eb15cc67725292e18169c7cc0dc7e6ba83/src/Tribe/Attendee_Registration/Modal.php#L15